charts/opensearch-dashboards/values.yaml

```diff
@@ -67,6 +67,9 @@ secretMounts: []
 
 podAnnotations: {}
 
+# Deployment annotations
+annotations: {}
```

Comment:
Hey Can we please use as `dashboardAnnotations`?

Reply:
Addressed this comment as well